tests/components/radial-tests.js

```diff
@@ -28,46 +28,42 @@ test('RadialChart: Basic rendering', t => {
   const pieSlices = $.find('.rv-radial-chart__series--pie__slice').length;
   t.equal(pieSlices, RADIAL_PROPS.data.length, 'should find the same number of slices as data entries');
 
-  const overlaySlices = $.find('.rv-radial-chart__series--pie__slice-overlay').length;
-  t.equal(overlaySlices, RADIAL_PROPS.data.length, 'should find the right number of overlay slices');
-
-  const labels = $.find('.rv-radial-chart__series--pie-label-inner-wrapper').length;
+  const labels = $.find('.rv-xy-plot__series--label-text').length;
```

Comment:
This might be out of scope (since this occurs in many tests), but I find that it's better to export constants that relate to class names from the implementation, and then import them in the test. Otherwise, you'll have to update the class name in two places if it ever changes.